topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin_jetbrains_runtime.md

```diff
@@ -0,0 +1,126 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# JetBrains Runtime (JBR)
+
+<link-summary>The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR), which can be provided in multiple ways.</link-summary>
+
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="Beta_Status"/>
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="faq"/>
+
+The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR) for running IDE locally, testing, or performing other actions that require Java Runtime adjusted by JetBrains.
+
+For the best experience, it is recommended to rely on JetBrains Runtime bundled with the IntelliJ Platform used for development.
+If one is not available, i.e., when targeting IntelliJ Platform snapshot or nightly releases, there are other ways available for providing JetBrains Runtime in a required version or variant.
+
+## Bundled (default, recommended)
+
+IDE releases provided with JetBrains Download CDN are OS-specific and contain JetBrains Runtime (JBR) already bundled within the archive.
+This is the recommended way for developing plugin, as it comes in exactly the same version as JetBrains IDEs delivered to users.
+
+To use this approach, no extra steps are required but declare the IntelliJ Platform, see: [](tools_intellij_platform_gradle_plugin_dependencies_extension.md#target-platforms).
+
+## Obtained with IntelliJ Platform from Maven
+
+If the IntelliJ Platform is resolved with the IntelliJ Maven Repository instead of JetBrains Download CDN, it has no JetBrains Runtime (JBR) bundled.
+In such a case, it is necessary to provide it in the suitable version with the version read from the <path>[intellijPlatform]/dependencies.txt</path> file.
```

Comment:
`[intellijPlatform]/dependencies.txt` is unclear to me. Is it the IDE home directory? Maybe an example could help?

Reply:
It's the IDE home directory but in the most of cases, it's long, ugly and should not be directly referred.